app/code/core/Mage/Checkout/Block/Onepage/Success.php

```diff
@@ -43,6 +45,17 @@ public function canPrint()
         return $this->_getData('can_view_order');
     }
 
+    /**
+     * Get url for view order details
+     *
+     * @return string
+     * @deprecated after 1.4.0.1
```

Comment:
if this is necessary probably we should un-deprecate it?

Reply:
You are right!